pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-270_cardiacMalformationTabChanges.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-270_Conotruncal" author="pegah">
+        <sql>
+            SELECT @realm:= ddp_instance_id FROM ddp_instance WHERE instance_name='singular';
+            SELECT @orderNumber := order_number from field_settings where column_display = "PA-VSD (TOF anatomy)"
+            and ddp_instance_id = @realm and field_type = 'CONOTRUNCAL_GROUP' ;
+
+            update field_settings set order_number = order_number + 1
+            where field_settings_id in ( select field_settings_id from (select * from field_settings) as tmp
+            where field_type = 'CONOTRUNCAL_GROUP' and order_number >= @orderNumber) and field_settings_id &lt;&gt; 0;
+
+            INSERT INTO field_settings (ddp_instance_id, field_type, column_name, column_display, display_type, order_number)
+            SELECT ddp_instance_id , 'CONOTRUNCAL_GROUP', 'TOF_MAPCA' ,'TOF + pulmonary atresia with MAPCAs',
+            'CHECKBOX', @orderNumber from ddp_instance where instance_name = 'Singular';
+
+            SELECT @orderNumber := order_number from field_settings where column_display = "DORV, os"
+            and ddp_instance_id = @realm and field_type = 'CONOTRUNCAL_GROUP' ;
+
+            update field_settings set order_number = order_number + 1
+            where field_settings_id in ( select field_settings_id from (select * from field_settings) as tmp
+            where field_type = 'CONOTRUNCAL_GROUP' and order_number >= @orderNumber) and field_settings_id &lt;&gt; 0;
+
+            INSERT INTO field_settings (ddp_instance_id, field_type, column_name, column_display, display_type, order_number)
+            SELECT ddp_instance_id , 'CONOTRUNCAL_GROUP', 'DORV_UNCOMMITTED_VSD' ,'DORV (uncommitted VSD)',
+            'CHECKBOX', @orderNumber from ddp_instance where instance_name = 'Singular';
+
+        </sql>
+    </changeSet>
+    <changeSet id="PEPPER-270_LVOTO_GROUP" author="pegah">
+        <sql>
+            SELECT @realm:= ddp_instance_id FROM ddp_instance WHERE instance_name='singular';
+            SELECT @orderNumber := order_number from field_settings where column_display = "IAA, A" and ddp_instance_id = @realm
```

Comment:
does the mix of `"` and `'` in one query matter?

Reply:
No that one I am sure of, because of my broken keyword it happens a lot and doesn't change anything